### PR TITLE
Update/refresh displayString.properties 

### PR DESCRIPTION
**Pull request by Bayernatoor**
See [project](https://github.com/bisq-network/projects/issues/38) for details. Related to [PR4515](https://github.com/bisq-network/bisq/pull/4515)

Corrected typos and grammatical errors. Improved some strings to help with clarity and guide users.

### CODE REVIEW

File: core/src/main/resources/i18n/displayStrings.properties
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
password.walletEncryptionFailed=Wallet password could not be set. You may have imported seed words which do not match the wallet database. Please contact the developers on Keybase (https://keybase.io/team/bisq).
```
I don't like the quotes, but this might be subjective.

File: core/src/main/resources/i18n/displayStrings.properties
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
  It is highly recommended that you make a backup of the application directory and write down your seed words before setting a password!
```

File: core/src/main/resources/i18n/displayStrings.properties
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
To open the emergency tool press \"Alt+e\" or \"Cmd/Ctrl+e\".
```
To be consistent with the way key shortcuts have been written elsewhere (not entirely consistent yet, but would like to make it that way).

File: core/src/main/resources/i18n/displayStrings.properties
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
  ● In case of a dispute the mediator or arbitrator cannot be of much assistance as it is usually difficult to get tamper-proof evidence \
```

**Review by m52go — COMMENTED** [review]
Good improvements—all small suggestions.

File: core/src/main/resources/i18n/displayStrings.properties
Comment (ghost) [reply core/src/main/resources/i18n/displayStrings.properties]: Enclosing the URL as follows will make it clickable, `[HYPERLINK:https://keybase.io/team/bisq]`
There are other example in displayStrings.properties.

Comment (m52go) [reply core/src/main/resources/i18n/displayStrings.properties]: Wonderful! Not sure how I missed that.

**Comment by Bayernatoor** [comment]
Thanks for the suggestions - will review and update :)

File: core/src/main/resources/i18n/displayStrings.properties
Comment (Bayernatoor) [reply core/src/main/resources/i18n/displayStrings.properties]: Ya not sure why I used quotes there. Thanks!

File: core/src/main/resources/i18n/displayStrings.properties
Comment (Bayernatoor) [reply core/src/main/resources/i18n/displayStrings.properties]: Ah yes good catch!

File: core/src/main/resources/i18n/displayStrings.properties
Comment (m52go) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
password.walletEncryptionFailed=Wallet password could not be set. You may have imported seed words which do not match the wallet database. Please contact the developers on Keybase ([HYPERLINK:https://keybase.io/team/bisq]).
```

**Review by Bayernatoor — COMMENTED** [review]
Looks good to me now! Thanks @m52go

**Comment by sqrrm** [comment]
Waiting for ack from @m52go before merging

**Comment by m52go** [comment]
ACK

**Review by sqrrm — APPROVED** [review]

**Comment by ripcurlx** [comment]
@Bayernatoor could you please force push to this branch again. It seems travis never run it as I didn't find it in the travis log.

**Comment by Bayernatoor** [comment]
> @Bayernatoor could you please force push to this branch again. It seems travis never run it as I didn't find it in the travis log.

@ripcurlx Hey sorry for the delay. How do go about doing that? does it have to been locally on my own fork? Do I need to update my repo first?

**Comment by Bayernatoor** [comment]
M52go walked me through it. 

I `git pull origin correctdisplaystrings`
and `git push -f origin correctdisplaystrings`

**Comment by ripcurlx** [comment]
> M52go walked me through it.
> 
> I `git pull origin correctdisplaystrings`
> and `git push -f origin correctdisplaystrings`

I have the feeling Travis flagged you or at least this PR. I'll cherry-pick the changes and try to create a PR myself.

**Comment by ripcurlx** [comment]
Superseded by #4750

**Pull request closed without merging**
